### PR TITLE
add PRDC

### PR DESCRIPTION
**Pull request by liyc-ai**
Add Implementation of PRDC, arXiv link: https://arxiv.org/abs/2306.06569

### CODE REVIEW

**Review by takuseno — CHANGES_REQUESTED** [review]
Thank you for your PR! This looked like a nice addition to d3rlpy :smile: I found a dependency problem as I commented. Please take a look at that. There might be trivial code changes that I would add. But, that's my job after we merge this PR. Thanks again!

File: setup.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
             "colorama",
             "dataclasses-json",
             "gymnasium>=1.0.0",
+            "scipy",
```
[the remark this answers is not in the document]
Comment (liyc-ai) [reply setup.py]: Thanks to your careful review. I have made the corresponding changes.

[the remark this answers is not in the document]
Comment (liyc-ai) [reply setup.py]: btw, I've also fix errors from `./scripts/lint`.

**Review by takuseno — APPROVED** [review]
LGTM. Thank you for the update!

**Pull request merged**